web/src/user_group_edit_members.js

```diff
@@ -112,17 +112,15 @@ function show_user_group_membership_request_result({
     message,
     add_class,
     remove_class,
-    subscribed_users,
-    already_subscribed_users,
+    already_added_users,
     ignored_deactivated_users,
 }) {
     const $user_group_subscription_req_result_elem = $(
         ".user_group_subscription_request_result",
     ).expectOne();
-    const html = render_user_group_subscription_request_result({
+    const html = render_user_group_membership_request_result({
         message,
-        subscribed_users,
```

Comment:
Removed this because we do not use this parameter as we do not show the list of users who were successfully added. We can add it later if needed.

Reply:
We probably should show that -- it's not obvious what they are in the case that you entered a stream or group name.

(Though we should keep in mind we'll generally be pushing folks subscribe a group to a group instead of adding its membership, I feel like there will still be cases where one wants to start by copying another group and then removing some users).